k8s/examples/wordpress/wordpress/25-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1beta1 
+kind: Ingress
+metadata:
+  name: test-ingress
+  labels:
+    app: wordpress
+    tier: frontend
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    # Enable client certificate authentication
+    #nginx.ingress.kubernetes.io/auth-tls-verify-client: "on"
+    # Create the secret containing the trusted ca certificates
+    #nginx.ingress.kubernetes.io/auth-tls-secret: "default/tls-certificate"
+spec:
+  tls:
+    - hosts:
+      - test.example.com
+      secretName: tls-certificate 
+  rules:
+    - host: test.example.com
+      http:
+        paths:
+        - path: /
+          backend:
+            serviceName: wordpress
+            servicePort: 8080
```

Comment:
Why is this pointing to 8080 if  the container port of the Wp app is serving from the 80? It can work, ofc, but why changing ports that are fine by default?